src/AddTodoForm.js

```diff
@@ -20,16 +21,14 @@ function AddTodoForm(props) {
 
     return (
         <form onSubmit={handleAddTodo}>
-            <label htmlFor="todoTitle" id="todoTitle">
-                Title
-            </label>
-            <input
-                type="text"
-                id="todoTitle"
-                name="title"
-                value ={todoTitle}
-                onChange = {handleTitleChange}
-            ></input>
+            <InputWithLabel
+            id = "id"
```

Comment:
props shouldn't have spaces. This should be id="id" and the others shouldn't have spaces as well.